src/pages/settings/tabs/currencies/edit-currency.vue

```diff
@@ -28,9 +30,21 @@
           :disabled="deletionDisabled"
           @click="onDeleteHandler"
         >
-          Delete
+          Delete currency
         </ui-button>
       </ui-tooltip>
+      <div class="edit-currency__checkbox">
+            <label :for="'checkbox-' + currency.id.toString()" class="edit-currency__label">
+              <span class="live-span">Live update</span>
+              <input
+                :id="'checkbox-' + currency.id.toString()"
```

Comment:
You don't need to specify `for` and `id` for `label > input` construction. By default, the label will always focus on nested input. Also for the future, use template strings for `id`-stuff, so `checkbox-${currency.id}` instead of `'checkbox-' + currency.id.toString()`.